uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/manager-leave/components/manager-leave-requests.js

```diff
@@ -359,12 +358,14 @@ define([
      */
     function loadLeaveRequest(type) {
       var filterByStatus = type !== 'filter',
-        pagination = type === 'filter' ? {} : vm.pagination,
+        pagination = type === 'filter' ? { size: 0 } : vm.pagination,
```

Comment:
Is this added because we want to load more than 25 objects? Please leave a comment on the code explaining what `size:0` is.